test/tests/outputs/json/distributed/tests

```diff
@@ -10,8 +10,16 @@
       check_files = 'distributed_out.json distributed_out.json.1'
       file_expect_out = '"part":'
       min_parallel = 2
-
-      detail = "that produces multiple files for parallel distributed data;"
+      max_parallel = 9 # file names get padded with zero above this
+      detail = "that produces multiple files for parallel distributed data for less than 10 and"
+    []
+    [default_more_mpi]
+      type = CheckFiles
+      input = distributed.i
+      check_files = 'distributed_out.json distributed_out.json.01 distributed_out.json.09'
```

Comment:
I want to see all the files between 1 and 9 too! :laughing: Jk

Reply:
I had to remove this, the checkpoint files in --recover were causing problems. I just limited the original test.